include/kllvm/ast/AST.h

```diff
@@ -182,6 +182,11 @@ class KOREObjectSymbol : public KORESymbol {
   bool isPolymorphic() const;
   bool isBuiltin() const;
 
+  /* instantiates this symbol (which should be parsed from a pattern in an axiom)
+     with the sorts corresponding to its actual sort parameters after instantiating
+     polymorphic parameters. This happens by replacing the variables in the arguments
+     of the specified declaration with their substitution in the arguments to the pattern
+     that were parsed in braces. */
```

Comment:
I'm still confused here. Where in KoreSymbol is that information from parsing stored? I guessed `arguments` are the sorts of the expected pattern arguments, but then where is information on the formal and actual sort parameters stored? Maybe this calls for some class-level comments.